modules/plugins/ui/breadcrumbs/breadcrumbs.nix

```diff
@@ -43,17 +45,33 @@ in {
       '';
     };
 
-    # maybe this should be an option to *disable* alwaysRender optionally but oh well
-    # too late
-    alwaysRender = mkOption {
-      type = bool;
-      default = true;
-      description = "Whether to always display the breadcrumbs component on winbar (always renders winbar)";
+    # Options for configuring Lualine integration of nvim-navic
+    lualine.winbar = {
+      enable =
+        mkEnableOption ''
+          automatically configuring a winbar component for Lualine.
+
+          ::: {.note}
+          This is **set to `true` by default**, which means nvim-navic
+          will occupy `winbar.lualine_c` for the breadcrumbs feature
+          unless this option is set to `false`.
+          :::
+        ''
+        // {default = true;}; # for retaining previous behaviour
+
+      alwaysRender = mkOption {
```

Comment:
Could use mkEnableOption here too

Reply:
Could, I just didn't want to deal with "Whether to enable..." suffix. It's hard to make sense with.